src/main/java/bookstore/dto/order/UpdateOrderDto.java

```diff
@@ -0,0 +1,10 @@
+package bookstore.dto.order;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+
+@Data
+public class UpdateOrderDto {
+    @NotBlank
+    private String status;
```

Comment:
you can easily accept a enum here, no need to parse it later

Reply:
thank you